FindMe/Location/LocationManager.m

```diff
@@ -94,7 +88,23 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
     if (locations.count >= 1) {
     }
     else {
-        
+    }
+}
+
+- (void)broadcastNotification
```

Comment:
Why notifications instead of the delegate pattern?

Reply:
Was trying to experiment with notifications but I will change it to delegate pattern